src/vos/vos_dtx.c

```diff
@@ -1101,3 +1627,263 @@ vos_dtx_mark_sync(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch)
 	vos_obj_release(occ, obj, false);
 	return rc;
 }
+
+/*
+ * \return	+2 if current DTX blob is batched cleanup.
+ *		+1 if current DTX entry has ever been committed.
+ *		Zero if the DTX entry is not committed.
+ *		Negative value if error.
+ */
+static int
+dtx_act_handle_batched_cleanup(struct vos_container *cont,
+			       struct dtx_batched_cleanup_blob *bcb,
+			       struct vos_dtx_act_ent_df *dae_df,
+			       umem_off_t *next_dsb)
```

Comment:
I struggled to understand this function and it could be helpful to add description for this function

Reply:
OK, I will add comment for the function.